utils/bazel/llvm-project-overlay/mlir/test/BUILD.bazel

```diff
@@ -552,7 +552,7 @@ cc_library(
 cc_library(
     name = "TestTransforms",
     srcs = glob(["lib/Transforms/*.cpp"]),
-    defines = ["MLIR_CUDA_CONVERSIONS_ENABLED"],
+    defines = ["MLIR_ENABLE_CUDA_CONVERSIONS"],
```

Comment:
There is a discrepancy between the change in Cmake and Bazel: shouldn't remove this and  rely on mlir-config.h  instead?

Reply:
I need to check this tomorrow at work but my guess was that this is activated by force here for testing. Not 100% sure now if that (1) is true and (2) makes sense.

Reply:
No I think until now this matching CMake: it was defined on the command line. But now that we move it to the config.h, you removed it from the CMake definitions so I would think we should match this in bazel

Reply:
Yeah, I think you are right. The last commit makes that change.